emails/2022-07-29.md

```diff
@@ -0,0 +1,23 @@
+It's time for [DoltHub's](https://www.dolthub.com) CEO weekly update. 
```

Comment:
I don't love this. I liked your more personal note from last week.

Reply:
Better?